code/modules/surgery/surgery_step.dm

```diff
@@ -93,7 +93,10 @@
 	var/was_sleeping = (target.stat != DEAD && target.IsSleeping())
 
 	// Skyrat Edit Addition - reward for doing surgery on calm patients, and for using surgery rooms(ie. surgerying alone)
-	if(was_sleeping || HAS_TRAIT(target, TRAIT_NUMBED) || target.stat == DEAD)
+	if(!(was_sleeping || HAS_TRAIT(target, TRAIT_NUMBED) || target.stat >= UNCONSCIOUS))
+		modded_time /= SURGERY_SPEEDUP_AREA
```

Comment:
/= is a cursed operator

Reply:
I feel like this could just be something that does *= 2 instead of /= 0.5
It's be cleaner code, even if it's a separate define.